feat(panorama): add projection type constant

peppsac opened the pull request:
Add explicit projection type constants instead of relying on the ratio to deduce
the projection.

zarov reviewed (approved):
LGTM, way cleaner than the `equirectangular` and `ratio` properties, thanks !

peppsac commented:
Thanks for the approval. Can you wait a sec before merging, there's something I need to check before :)

peppsac commented:
I'll push a commit to fix a breakage from 9241818aadc21cd0ad71b73d6b4013b922c3da40: before this commit the panorama was oriented such as North = center of the image.

gchoqueux committed:
feat(panorama): add projection type constant #677

Add explicit projection type constants instead of relying on the ratio to deduce
the projection.
